a/activemq/Dockerfiles/latest_centos_7/run/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ppc64le/centos:7
```

Comment:
same comment as earlier about use of centos:7 and "ppc64le/"

Reply:
okay